.cache/325c8f456729b912b0d2134054eb7448-dfeeb2271cc2857eb0a45a5003c8bbee

```diff
@@ -0,0 +1 @@
+{"value":{"success":true,"data":{"latest":{"version":"4.0.0","info":{"plain":"- upgrade webpack & babel to latest\n- new addParameters and third argument to .add to pass data to addons\n- added the ability to theme storybook\n- improved ui for mobile devices\n- improved performance of addon-knobs"}}},"time":1543938430826},"type":"Object"}
```

Comment:
Maybe add a line in `.gitignore` to get rid of the whole `.cache` folder ... I'm not sure what it's for, but it probably shouldn't be checked in